src/SIIPExamples.jl

```diff
@@ -0,0 +1,87 @@
+module SIIPExamples
+
+export print_struct
+
+using Weave
+
+repo_directory = dirname(joinpath(@__DIR__))
+
+"""
+`weave_file(folder::AbstractString, file::AbstractString)`
+
+Checks if the file has been modified since the last Weave and updates the notebook and tests accordingly.
+
+* `folder` = Name of the folder the tutorial is in
+* `file` = Name of the tutorial file
+* `force` = foce weave irrespective of file changes
+"""
+function weave_file(folder, file; force = false)
+    
```

Comment:
I would write the above like this instead:

```
"""
    weave_file(folder::AbstractString, file::AbstractString)

Checks if the file has been modified since the last Weave and updates the notebook and tests accordingly.
* `folder` = Name of the folder the tutorial is in
* `file` = Name of the tutorial file
* `force` = foce weave irrespective of file changes
"""
function weave_file(folder, file; force = false)
```

i.e. use 4 spaces instead of single back tick quotes. In my opinion it reads better.